src/ui/components/DropdownMenu/index.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+import onClickOutside from 'react-onclickoutside';
+
+import Icon from 'ui/components/Icon';
+import DropdownMenuItem from 'ui/components/DropdownMenuItem';
+
+import './styles.scss';
+
+
+type Props = {|
+  text: string,
+  children: Array<DropdownMenuItem>,
+  className?: string,
+|};
+
+export class DropdownMenuBase extends React.Component {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { buttonIsActive: false };
+  }
+
+  state: {| buttonIsActive: bool |};
```

Comment:
Huh, nitpick but I thought we'd put these before `constructor`.

Reply:
nope^^

```
/Users/william/projects/mozilla/addons-frontend/src/ui/components/DropdownMenu/index.js
  20:3  error  state should be placed after constructor  react/sort-comp
```